app/views/application/_button_edit.html.slim

```diff
@@ -0,0 +1,3 @@
+.row
+  .col-xs-12
+    = link_to t_action(:edit), { controller: controller_name, action: :edit, id: params[:id] }, { class: 'btn btn-default' }
```

Comment:
Because this partial wants to be generic, we do not have edit_client_path( nor `@client`. Or we have it, but it could be `@client`, `@volunteer`, `@user`, .... depending on which show-view it is being rendered on.

This way we can just generate the correct path from the parameters and the responsible controller. It means that this partial would even work if we'd make a custom resource path, that isn't named edit_xyz_path.